Documentation/specs/runtime-configuration-file.md

```diff
@@ -25,6 +25,10 @@ The files are both JSON files stored in UTF-8 encoding. Below are sample files.
     "runtimeConfig": {
         "gcServer": true,
         "gcConcurrent": false
+    },
+    "targetFramework": {
```

Comment:
No we aren't doing this. We're going to use the package. There's no point duplicating that information. On the edge we'll put redist: true or something